custom/conf/app.example.ini

```diff
@@ -827,6 +827,14 @@ LEVEL = Info
 ;; Dependencies can be added from any repository where the user is granted access or only from the current repository depending on this setting.
 ;ALLOW_CROSS_REPOSITORY_DEPENDENCIES = true
 ;;
+;; Default map service. No external API support has been included. A service has to allow
+;; searching using URL parameters. Uncomment this option and leave it empty to disable this.
+;; Some examples are:
+;; - https://www.google.com/maps/place/
+;; - https://www.mapquest.com/search/
+;; - https://www.bing.com/maps?where1=
+; USER_LOCATION_QUERY_URL = https://www.openstreetmap.org/search?query=
```

Comment:
It's why I originally also had a boolean variable, but I understand that this may not work internationally. Deal.

Reply:
> It's why I originally also had a boolean variable,

IMO there is no difference , and this single option is simpler than two variables.

Before:

`ENABEL_xxx = false`  vs ``ENABEL_xxx = true + USER_LOCATION_QUERY_URL=.....`

Now:

`USER_LOCATION_QUERY_URL=`   vs `USER_LOCATION_QUERY_URL = .....`

Reply:
Do you think that it would be a good idea to add this during the instance installation flow so that users can select from a dropdown menu with a diverse set of options, or is it too much of a vanity feature?

Reply:
I think it's not necessary.

There are different users. Some private instance/enterprise instance users don't need it. I guess most users are this kind users.

Only some large public instances need it, while such instances needs a lot of fine tunes, the "install" page is too simple for them, so it doesn't help much in this case IMO.

Reply:
> Only some large public instances need it, while such instances needs a lot of fine tunes, the "install" page is too simple for them, so it doesn't help much in this case IMO.

That's fine, I do understand that the mindset I bring here is the mindset of a large public instance.

Reply:
Really sorry for all of those hectic changes and thanks for your extra feedback that helped make this feature much more universal.

Reply:
Never mind, thank you for your patience.

I tried to make some changes, if there is something wrong, don't hesitate to correct me.

Reply:
> I tried to make some changes, if there is something wrong, don't hesitate to correct me.

I just wanted to get an idea included upstream and made a lot of concessions to comply with the general idea of making even the smallest features usable by as many people as possible. I trust your judgement here, but thanks for the offer.

Reply:
Thank you and sorry for that feeling. Actually I didn't mean that and I appreciate you very much.

Sometimes I would edit some PRs directly (since the checkbox of "Allow Maintainers Edit" is enabled by default) because I worry about my discussion is not clear by text (well, Linus ever said "show me the code" 😄 ) which would might cause people tired or misunderstanding.

If you don't like the PR being edited, just tell me, or if anything is unclear or anything I need to do, also let me know.